src/Nordea/Components/InfoBanner.elm

```diff
@@ -0,0 +1,85 @@
+module Nordea.Components.InfoBanner exposing (Appearance(..), Config(..), bodyText, cross, view)
+
+import Css
+    exposing
+        ( alignItems
+        , auto
+        , backgroundColor
+        , border
+        , borderRadius
+        , displayFlex
+        , flexDirection
+        , flexShrink
+        , flexStart
+        , marginLeft
+        , marginRight
+        , num
+        , padding
+        , rem
+        , row
+        , width
+        )
+import Html.Styled as Html exposing (Attribute, Html)
+import Html.Styled.Attributes exposing (css)
+import Nordea.Components.Checkbox exposing (Appearance)
+import Nordea.Components.Text as Text
+import Nordea.Resources.Colors as Colors
+import Nordea.Resources.Icons as Icons
+import Nordea.Themes as Themes
+
+
+type Appearance
+    = Warning
+    | Info
+
+
+type Config msg
+    = Appearance Appearance
+
+
+view : List (Config msg) -> List (Attribute msg) -> List (Html msg) -> Html msg
+view config attrs children =
+    let
+        appearance =
+            config
+                |> List.filterMap
```

Comment:
could just be normal List.map i think?

Reply:
Trying to be an example of the new pattern